includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2747,7 +2747,7 @@ protected function get_account_branding_secondary_color( $default_value = '' ):
 	 *
 	 * @return string The domestic currency code.
 	 */
-	protected function get_account_domestic_currency(): string {
+	public function get_account_domestic_currency(): string {
```

Comment:
I changed this from `protected` to `public` so that it could be called in the `PaymentMethodsCompatibility` multi-currency class. I _think_ it's fine?

Reply:
I think it's fine, but overall, this method seems to be placed in the unexpected class. I'd expect it to be located somewhere in the `includes/class-wc-payments-account.php` near `get_account_country()` method :)